src/crypto_helpers.rs

```diff
@@ -1,15 +1,54 @@
 use nanos_sdk::bindings::*;
 use nanos_sdk::ecc::{CurvesId, DEREncodedECDSASignature};
 
-// TODO: make a macro that derives the path by itself
-// ex: make_bip32_path!(b"44'/535348'/0'/0/0";
-// =>
-pub const BIP32_PATH: [u32; 5] = [
-        0x8000002cu32,
-        0x80082b34, 
-        0x80000000, 
-        0x00000000, 
-        0x00000000];
+/// ASCII value of the character 0.
+const ZERO: u8 = 48;
+/// ASCII value of the character 9.
+const NINE: u8 = 57;
+
```

Comment:
Should we be avoiding const declaration because we're working with limited space on the device? I'm not sure what are the "best practices" here..